### DISCUSSION TITLE
Dependabot creates PRs on forks

### DISCUSSION THREAD

**Discussion by lewis-yeung** [JanDeDobbeleer/oh-my-posh#2846] [General]
Since [`dependabot.yml`](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/.github/dependabot.yml) was introduced, Dependabot is creating PRs on forks, which is so annoying. Related: dependabot/dependabot-core#2804. Unfortunately, this remains unresolved to this day. If we add a restriction `target-branch: "main"` for each package in `dependabot.yml`, and change the default branch name of a fork to something other than `main`, will it stop Dependabot from creating any PRs?

**Comment by JanDeDobbeleer** [comment]
@lewis-yeung yes, that should work. You could also adjust that in the fork's main tbh. The work is identical imho.

**Reply by lewis-yeung** [reply]
Seems to work. It says `Dependabot couldn't find the branch main` now. 😄

**Reply by lewis-yeung** [reply]
@JanDeDobbeleer And should also we add `target-branch: "main"` in this repo? If so, the default branch of a fork can still be easily synced with upstream.

**Reply by JanDeDobbeleer** [reply]
Sure, no harm in that
